### PR TITLE
#156577924 Send alerts via multiple methods e.g SMS

### PR DESCRIPTION
**Pull request by kasigazibrian**
### What does this PR do?
This PR enables an sms and telegram alert to be sent to a user to let them know about their check status. 

### Description of Task to be completed?
The user should be able to receive important information concerning their jobs through multiple channels like SMS and Telegram.

### How should this be manually tested?
1. Create a job with a timeout period of one minutes and grace period of one minute.
2.  Select the integrations tab and select SMS. 
3.  Follow instructions on how to integrate SMS
4.  Ping the job URL to activate it
5.  After that wait for the ping to go down
6.  An SMS alert should be sent to your phone number
7.  Do the same for telegram integration from step 2  to 5
8. A telegram text alert should be sent to your telegram application

### Any background context you want to provide?
Previously a user was not able to receive an alert about his/her check status using telegram and SMS. Now they can.

### What are the relevant pivotal tracker stories?
[#156577924](https://www.pivotaltracker.com/story/show/156577924)

### CODE REVIEW

**Review by Rwothoromo — COMMENTED** [review]
@kasigazibrian Good work; it looks good to me.
To increase test coverage, you could mock sms and telegram the same way `VictorOps` is tested in `hc/api/tests/test_notify.py` i.e. `test_victorops(self, mock_post)`

We could also consider including them in `hc/front/tests/test_add_channel.py` via `test_instructions_work`.

And also a screenshot of the integrations page in this PR

File: hc/front/views.py [hunk 1]
```diff
@@ -393,6 +401,31 @@ def add_email(request):
...
+
+        def start(bot, update):
+            bot.send_message(chat_id=update.message.chat_id,
+                             text=str("Hello, Welcome to healthchecksbot. " + " Your chat ID is " +
+                                      str(update.message.chat_id)))
+        start_handler = CommandHandler('start', start)
+        # add handlers
+        updater.dispatcher.add_handler(start_handler)
+        updater.start_polling()
+        return render(request, 'integrations/add_telegram.html', ctx)
```
Comment (Rwothoromo) [inline hc/front/views.py:419]: This return statement however, may not be necessary.

**Comment by louicoder** [comment]
Good work brian. just be mindful of extra lines from one assert statement to another

**Review by emugaya — APPROVED** [review]
lgtm

File: hc/front/views.py (hunk 1 above)
Comment (kasigazibrian) [reply hc/front/views.py:419]: Okay let me get rid of it

Comment (kasigazibrian) [reply hc/front/views.py:419]: Thank you for that feedback. I have implemented the changes

**Pull request merged**